README.md

```diff
@@ -192,6 +216,14 @@ limitations under the License.
 
 ## Release Notes
 
+### v1.3.0
+- Add the ability to assemble the legacy version of iLib.  
+  When the legacyilib flag is set, it assembles files in the legacy style, which refers to the files from iLib. The generated output includes separate files for JS code and locale data: one JS file containing the JS code and iLib's root locale data, and additional JSON files for locale-specific data. Currently, it generates files in the [language].js format.  
+e.g
+```
+ ilib-assemble output-dir --legacyilib --ilibPath ~/Source/develop/ --ilibincPath src/ilib-assemble-inc.js -l "af-ZA,am-ET,ko-KR"
+```
+
```

Comment:
Also add:
The end result is similar to the result of assembling ilib using webpack or rollup, but it is simpler and produces data files as javascript which can be used in other situations without requiring you to use webpack or rollup.